ProcessLib/LiquidFlow/LiquidFlowLocalAssembler.h

```diff
@@ -86,6 +86,12 @@ class LiquidFlowLocalAssembler : public LiquidFlowLocalAssemblerInterface
                   std::vector<double>& local_K_data,
                   std::vector<double>& local_b_data) override;
 
+    void coupling_assemble(double const t, std::vector<double> const& local_x,
```

Comment:
Sorry the name of ``coupling_assemble`` is not changed because it introduced in my last merged PR, and it exists in  the base class of the local assembler class as well. If necessary, I will change it to ``assembleWithCoupling`` in another PR.